en/docs/guides/password-mgt/recover-password.md

```diff
@@ -1,87 +1,110 @@
-# Recover Password 
+# Recover Password via Email
 
-## Recover password using the user portal 
+WSO2 Identity Server enables resetting user passwords by emailing a password reset link to the user’s registered email Id.
 
-{!TODO:insert-fragment!} 
+---
+
+## Prerequisites
+
+**Create a user**
+
+{!fragments/create-user-for-recovery-flows.md!}
+
+**Configure the email adapter to send emails**
+
+Enable the email sending configurations of the WSO2 Identity Server as explained here.
+
+{!fragments/configure-email-sending.md!}
 
 ---
+    
+## Enable password reset via email
 
-## Recover password using the REST API
+Follow the steps below to configure WSO2 Identity Server to enable password reset via email notifications.  
 
-You can use the following CURL command to recover a password using REST API. 
+1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory and check whether the following listener 
+    configs are in place.
+    
+    ```
+    [event.default_listener.identity_mgt]
+	priority= "50"
+	enable = false
+	[event.default_listener.governance_identity_mgt]
+	priority= "95"
+	enable = true
+	[event.default_listener.governance_identity_store]
+	priority= "97"
+	enable = true
+	```
 
-### Send recovery notification
+	!!! info 
+        This is already configured this way by default. You can skip this step if you have not changed this configuration previously.
```

Comment:
Please check if this alignment appears properly.

Reply:
yes it appears properly.